pkg/operator/ceph/disruption/register_controllers.go

```diff
@@ -16,25 +16,36 @@ limitations under the License.
 
 // Package controllers contains all the controller-runtime controllers and
 // exports
-package controllers
+package disruption
 
 import (
 	"fmt"
 
-	"github.com/rook/rook/pkg/operator/ceph/disruption/controllerconfig"
-
 	"github.com/rook/rook/pkg/operator/ceph/disruption/clusterdisruption"
+	"github.com/rook/rook/pkg/operator/ceph/disruption/controllerconfig"
+	"github.com/rook/rook/pkg/operator/ceph/disruption/machinedisruption"
+	"github.com/rook/rook/pkg/operator/ceph/disruption/machinelabel"
 	"github.com/rook/rook/pkg/operator/ceph/disruption/nodedrain"
 
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 )
 
+var (
+	EnableFencing bool
```

Comment:
Let's not use the term "Fencing" to refer to "machine disruption budgets" so we stay consistent with Kubernetes terminology.

Reply:
Update: I was reading `machine` disruption budget as the same as Kubernetes' `pod` disruption budget.

Reply:
It really is about enabling the machinedisruptionbudget controller, so i'll change it at this level, even if we still refer to fencing in comments or elsewhere.